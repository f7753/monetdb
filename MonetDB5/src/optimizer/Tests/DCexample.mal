    V7 := bbp.new(:void,:int);
    V10 := bbp.new(:int,:void);
    V16 := algebra.markH(V7);
    V17 := algebra.join(V16,V7);
    V19 := bbp.new(:void,:int);
    V22 := bbp.new(:void,:int);
    V23 := algebra.join(V16,V22);
	io.print("done");

optimizer.deadCodeRemoval();
mdb.list();
