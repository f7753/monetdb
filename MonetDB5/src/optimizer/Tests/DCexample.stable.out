stdout of test 'DCexample` in directory 'src/optimizer` itself:


# 16:14:06 >  
# 16:14:06 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/log" --set mapi_port=31571 --set sql_port=47454 --set xquery_port=51001 --set monet_prompt= --trace --dbname=mTests_src_optimizer  DCexample.mal </dev/null
# 16:14:06 >  

# Monet Database Server V4.99.19
# Copyright (c) 1993-2005, CWI. All rights reserved.
# Compiled for x86_64-redhat-linux-gnu/64bit with 64bit OIDs; dynamically linked.
# config:/ufs/mk/monet5/src/mal/Tests/All.conf
# dbfarm:/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/dbfarm
# dbname:mTests_src_optimizer
# Visit http://monetdb.cwi.nl/ for further information.
#    V7 := bbp.new(:void,:int);
#    V10 := bbp.new(:int,:void);
#    V16 := algebra.markH(V7);
#    V17 := algebra.join(V16,V7);
#    V19 := bbp.new(:void,:int);
#    V22 := bbp.new(:void,:int);
#    V23 := algebra.join(V16,V22);
#	io.print("done");
#
#optimizer.deadCodeRemoval();
#mdb.showFunction();
#
function user.main():int;	# 0  (main:int)
    io.print("done");	# 1 IOprint_val ($10:int)<-($11:str)
end main;	# 2  
[ "done" ]

# 16:14:06 >  
# 16:14:06 >  Done.
# 16:14:06 >  

